app/src/main/java/com/flexcode/yummy/data/local/dao/CategoriesDao.kt

```diff
@@ -0,0 +1,20 @@
+package com.flexcode.yummy.data.local.dao
```

Comment:
This is great there was a categories dao in the meals dao maybe you can delete the queries in the meals dao relating to categories so that we use the one you created

Reply:
working on it